k8s/namespaces/private-law/prl-citizen-frontend/prl-citizen-frontend.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: prl-citizen-frontend
+  namespace: private-law
+spec:
+  releaseName: prl-citizen-frontend
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/prl-citizen-frontend
+  values:
+    java:
+      replicas: 2
+      memoryRequests: "768Mi"
+      cpuRequests: "1000m"
+      memoryLimits: "3072Mi"
+      cpuLimits: "2500m"
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 15
```

Comment:
is there something wrong with the default? new apps should generally work OOTB

Reply:
addressed and removed. default will be followed